cpp/models/abm/parameters.h

```diff
@@ -308,19 +308,34 @@ struct HighViralLoadProtectionFactor {
     }
 };
 
+/** 
+ * @brief The index for all the testing types.
+ * Can be used as 0-based index.
+ */
+enum class TestingTypeIndex : std::uint32_t
+{
+    GenericTest = 0,
+    AntigenTest = 1,
+    PCRTest     = 2,
+    Count //last!!
+};
+
 /**
  * @brief Parameters that describe the reliability of a test.
  */
 struct TestParameters {
-     UncertainValue<> sensitivity;
-     UncertainValue<> specificity;
+    UncertainValue<> sensitivity;
+    UncertainValue<> specificity;
+    TimeSpan required_time;
+    TimeSpan validity_period;
```

Comment:
Yes and no. I think it could be grouped here with other parameters and you can modify it when initialising the TestingScheme as well.  
For an initial value, I guess 1 day should be ok but we might need to discuss a bit with the team.